docs/syntax/scss.md

```diff
@@ -278,7 +278,23 @@ Modules that make use of styles defined in other modules that support silent mod
 		@include oAnotherThingFoo();
 	}
 
-Finally, in documentation, modules *must* provide information about both silent and non-silent methods, where supported, and must put the default first (i.e. if silent mode is by default **on**, the module must document the silent mode integration first).
+When a module that supports silent mode is used with silent mode turned off (i.e. `$o-{modulename}-is-silent: false`), after the module has output the CSS, it *must* also reset the value of `$o-{modulename}-is-silent` to true, (this prevents the accidental output of styles if the module is included a second time):
```

Comment:
This feels like a really fragile solution to me.  I mean, if Next's components A and B both do something like this:

```
$o-doobury-is-silent: false;
@import(o-doobury);
```

and o-doobury does this:
```
.o-doobury__ojemeflip { ... }
$o-doobury-is-silent: true;
```

But then a Next service imports both those Next components, the styles are still going to be output twice.  The only solution I can see is don't unsilence components in a component that may itself be a dependency.  Am I missing something?